cms/static/js/factories/login.js

```diff
@@ -12,6 +12,15 @@ define(['jquery.cookie', 'utility'], function() {
             });
         }
 
+        // Clear the login error message when credentials are edited
+        $('input#email').on('input',function() {
+            $('#login_error').removeClass('is-shown');
```

Comment:
@sarina: we have an `is-hidden` CSS class that sets `display: none`: https://github.com/edx/edx-platform/blob/master/common/static/sass/_mixins.scss#L342-L344 That seems like a good way to make the error message disappear -- that, or calling [`.hide()`](http://api.jquery.com/hide/) on the element via jQuery. Simply removing the `is-shown` CSS class will not actually hide the error message, unless there's some other CSS/Javascript interaction going on that I'm not aware of.